gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilExtension.kt

```diff
@@ -1,6 +1,8 @@
 package com.squareup.anvil.plugin
 
-open class AnvilExtension {
+import org.gradle.api.provider.Property
+
+abstract class AnvilExtension {
```

Comment:
in theory this is a potentially breaking change, but I don't think it's expected for folks to manually instantiate this

Reply:
I agree, that should really be considered unsupported, even if it was technically possible.